### PR TITLE
added shopcarts/users

### PR DESCRIPTION
**Pull request by heejungRyu**
I added the function for the url shopcarts/users

It uses get method and requires a parameter amount which is a number.

### CODE REVIEW

File: app/service.py [hunk 1]
```diff
@@ -309,6 +317,56 @@ def post(self):
...
+        Get the shopcart list of users
+        This endpoint will show the list of the user shopcart list
+        with amount more than given amount
+        """
+        amount = request.args.get('amount');
+        app.logger.info("Request to get the list of the user shopcart having more than {}".format(amount))
+        
+        if amount is None:
+            app.logger.info("amount is none")
+            raise NotFound("Required parameter amount not found")
```
Comment (conychou) [inline app/service.py]: should it be 
abort(status.HTTP_404_NOT_FOUND, 'Required parameter amount not found') ?

File: app/service.py [hunk 2]
```diff
@@ -309,6 +317,56 @@ def post(self):
...
+        else:
+            try:
+                app.logger.info("try start")
+                amount = float(amount)
+                app.logger.info("try end")
+            except ValueError:
+                app.logger.info("value error")
+                abort(status.HTTP_400_BAD_REQUEST, 'parameter is not valid: {}'.format(amount))
+                #raise BadRequest("Required parameter 'amount' should be a float")
```
Comment (conychou) [inline app/service.py]: please just remove the unused part

File: app/service.py [hunk 3]
```diff
@@ -309,6 +317,56 @@ def post(self):
...
+        shopcarts = Shopcart.find_users_by_shopcart_amount(amount)
```
[the remark this answers is not in the document]
Comment (heejungRyu) [reply app/service.py]: Yes this is the function looking for users who has items in the shopcart amount more than a given amount.

File: app/service.py (hunk 2 above)
Comment (heejungRyu) [reply app/service.py]: Done

File: app/service.py [hunk 4]
```diff
@@ -309,6 +317,56 @@ def post(self):
...
+    GET /users - Returns the list of the users whose shopcarts' total is more than a certain amount
+    """
+    #@api.expect(amount_fields)
+    @ns.param('amount', 'amount for searching')
+
+    ############################################################################
+    # QUERY DATABASE FOR SHOPCARTS HAVING PRODUCTS WORTH MORE THAN GIVEN AMOUNT
+    ############################################################################
+    @ns.doc('get_users_with_shopcart_more_than_given_amount')
+    #@ns.marshal_list_with(string)
```
Comment (conychou) [inline app/service.py]: @ns.response(404, 'amount is none')

File: app/service.py (hunk 1 above)
Comment (heejungRyu) [reply app/service.py]: got the point. done.

**Pull request merged**